substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/phases/SharedGraphBuilderPhase.java

```diff
@@ -189,9 +188,8 @@ protected void handleIllegalNewInstance(JavaType type) {
             if (allowIncompleteClassPath) {
```

Comment:
The comment above the `if` refers to the removed `--allow-incomplete-classpath` option

Reply:
Not really. Now the value of `allowIncompleteClassPath` here is determined per method that gets parsed via `!LinkAtBuildTimeSupport.singleton().linkAtBuildTime(method.getDeclaringClass())` in the `SharedBytecodeParser` constructor.